### ISSUE TITLE
weight decay

### ISSUE THREAD

**Issue by rwightman** [google/automl#567]
@mingxingtan  Curious if there was any search / experimentation on the weight decay and impact on training results, have any values besides 4e-5 been tried, especially with longer than 300 epochs, such as the 600 epoch training for D7?

**Comment by mingxingtan** [comment]
Good question, @rwightman 

I tuned weight decay for D0 (epoch 120) based on candidate values {1e-5, 2e-5, 4e-5, 8e-5} and then fix it to 4e-5 for all other models.  It is probably not optimal for bigger models or different training settings.

**Comment by rwightman** [comment]
k, thanks, if I try anything different with a positive result I'll mention it here, can close this unless you want to keep it as a reference

**Comment by glenn-jocher** [comment]
@mingxingtan @rwightman the generalizability of training settings across the model space is difficult to predict. In my experience larger models will overfit the same dataset more quickly, all else equal, so may benefit from a more aggressive normalization or weight decay strategy.

I don't have a good sense for how this translates to fintetuning pretrained models however, where overfitting early is inevitable and perhaps even desirable.
